clouds/clouds-aws/src/test/java/aws/cloudformation/di/AwsClientsTests.java

```diff
@@ -0,0 +1,62 @@
+package aws.cloudformation.di;
+
+
+import aws.core.identification.AwsAutoResolve;
+import aws.core.identification.AwsRegion;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import software.amazon.awssdk.services.ec2.Ec2Client;
+import software.amazon.awssdk.services.s3.S3Client;
+import sunstone.core.SunstoneExtension;
+
+import static aws.cloudformation.AwsTestConstants.region;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(SunstoneExtension.class)
+public class AwsClientsTests {
+
+    @AwsRegion(region)
```

Comment:
Why there is need to introduce new annotation for type `Ec2Client` ?

Now we have 2 annotationts to get `Ec2Client`. Do we have something like  that for Azure as well?

Cant have `@AwsAutoResolve` annotation `region` parameter?

Reply:
Well, I suppose AutoResolve annotation may contain region parameter as it is very general and many resources are in some region.